apps/api/src/domains/admin/feedback/feedback.os.service.ts

```
@@ -480,12 +495,15 @@ export class FeedbackOSService {
-      if (queries?.[i].issueIds) {
+      if (queries?.[i].key === 'issueIds') {
+        const condition = queries[i].condition;
+
         const feedbackIds = await this.issueIdsToFeedbackIds(
-          queries[i].issueIds as number[],
+          queries[i].value as number[],
+          condition,
         );
 
-        delete queries[i].issueIds;
+        delete queries[i].value;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)